packages/generators/utils/checkDefaults.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ *
+ * Utility function to check if the user config is same as default.
+ *
+ * @param	{Object[]} initConfig 	- config generated by the user.
+ * @returns {Boolean} - Whether the default config is used or not.
+ */
+
+export default function checkDefaults(initConfig: Record<string, any>): boolean {
+	// This is the default prompts selections generated on picking default options.
+	const defaultConfig = {
+		babelConfirm: true,
```

Comment:
If you agree, I think it would be nice to have those as constants so that we can use them as default values while prompting

Reply:
Yeah would be much cleaner that way, how about a `constants` file in the utils folder? From where we can export all constants? 🤔

Reply:
Sound good